docs/source/start.rst

```diff
@@ -13,9 +13,9 @@ take you through the process of installing all three of these. We also step you
 
 Upgrading or Installing pyQuil
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-PyQuil 2.0 is our library for generating and executing Quil programs on the Rigetti Forest platform.
+pyQuil 2.0 is our library for generating and executing Quil programs on the Rigetti Forest platform.
```

Comment:
pyQuil 3.0? Maybe just drop the version here.

Reply:
Good catch. Meant to drop the version here. Will do.